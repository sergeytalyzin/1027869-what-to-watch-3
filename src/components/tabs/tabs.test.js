import React from "react";
import renderer from "react-test-renderer";
import Tabs from "./tabs.jsx";


it(`MoviePageDetails is rendered correctly`, () => {
  const tree = renderer.create(<Tabs
    activeTab={`overview`}
    handleClickTab={()=>{}}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
