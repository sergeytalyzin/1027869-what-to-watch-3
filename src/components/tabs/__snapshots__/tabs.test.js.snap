// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePageDetails is rendered correctly 1`] = `
<ul
  className="movie-nav__list"
>
  <li
    className="movie-nav__item false"
    onClick={[Function]}
  >
    <a
      className="movie-nav__link"
      href="#"
    >
      Overview
    </a>
  </li>
  <li
    className="movie-nav__item false"
    onClick={[Function]}
  >
    <a
      className="movie-nav__link"
      href="#"
    >
      Details
    </a>
  </li>
  <li
    className="movie-nav__item false"
    onClick={[Function]}
  >
    <a
      className="movie-nav__link"
      href="#"
    >
      Reviews
    </a>
  </li>
</ul>
`;
